lib/assessment.rb

```diff
@@ -5,35 +5,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
```

Comment:
The initialize thing?


On 10/23/15 7:06 PM, "Jeffrey Horn" <notifications@github.com> wrote:

> In lib/assessment.rb
> <https://github.com/ga-wdi-boston/ruby-objects-assessment/pull/43#discussion_r
> 42923545> :
>> > @@ -5,35 +5,47 @@
>> >  # Create a class called Dog. It should have, as attributes, name, breed,
>> and age.
>> >  # Make all of those attributes read-only.
>> >  
>> > -##
>> > -# your code here
>> > -##
>> > +class Dog
>> > +  attr_reader :name
>> > +  attr_reader :breed
>> > +  attr_reader :age
> This class is incomplete. What are you missing?
> 
> 
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/ruby-objects-assessment/pull/43/files#r42923
> 545> .
> 
>   
> 

-- 
³To avoid criticism, do nothing, say nothing, and be nothing.² --Elbert
Hubbard